src/natcap/invest/ndr/ndr.py

```diff
@@ -1366,10 +1380,12 @@ def _aggregate_and_pickle_total(
 
 def create_vector_copy(base_vector_path, target_vector_path):
     """Create a copy of base vector."""
-    if os.path.isfile(target_vector_path):
+    if os.path.exists(target_vector_path):
         os.remove(target_vector_path)
-    base_vector = gdal.OpenEx(base_vector_path, gdal.OF_VECTOR)
-    driver = gdal.GetDriverByName('ESRI Shapefile')
-    target_vector = driver.CreateCopy(
-        target_vector_path, base_vector)
-    target_vector = None  # seemingly uncessary but gdal seems to like it.
+
+    base_wkt = pygeoprocessing.get_vector_info(
+        base_vector_path)['projection_wkt']
+    # use reproject_vector to create a copy in geopackage format
+    # keeping the original projection
+    pygeoprocessing.reproject_vector(
+        base_vector_path, base_wkt, target_vector_path, driver_name='GPKG')
```

Comment:
I'm curious if `CreateCopy` doesn't work for GPKG drivers, or if there was another reason to piggy back off of `reproject_vector` here?

Reply:
`CreateCopy` raised an error when I tried to convert shapefile -> gpkg. I suspect it's not really meant for file conversion.

Reply:
Would be nice to have a pygeoprocessing function specifically for copying vectors.

Reply:
I believe we did have some functions like that back in a much earlier API implementation. I certainly remember writing a lot of vector copying, clipping, and intersection code when the gdal Python bindings were a lot less robust. When James and Rich refactored that to the modern day pygeoprocessing, I think a lot of that stuff was left behind.